src/layout/core/borders/borders.rs

```diff
@@ -96,7 +111,16 @@ impl Renderable for Borders {
     fn reallocate_buffer(mut self, mut geometry: Geometry) -> Option<Self>{
         // Add the thickness to the geometry.
         let thickness = Borders::thickness();
-        let title_size = Borders::title_bar_size();
+
+        let title_count = match (self.layout, &self.children) {
+            (Some(Layout::Stacked), &Some(ref children)) =>
```

Comment:
You can remove this slightly perplexing looking `&` on the option to avoid moving the value by doing this (and removing the ref too as an added bonus):

```rust
        let title_count = match (self.layout, self.children.as_ref()) {
            (Some(Layout::Stacked), Some(children)) =>
```